bucket/electron-fiddle.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "0.26.0",
+    "description": "The easiest way to get started with Electron",
```

Comment:
Marketing text which does not mean anything to user as it does not describe the application.

Reply:
I googled and replaced the description. Also it stay same in [homebrew](https://github.com/minonl/homebrew-cask/blob/040dffb345b5c48688c5707311a8bfbe9fe9e36d/Casks/electron-fiddle.rb#L17)